h2o-py/h2o/frame.py

```diff
@@ -1814,9 +1814,21 @@ def impute(self, column=-1, method="mean", combine_method="interpolate", by=None
         if is_type(column, str): column = self.names.index(column)
         if is_type(by, str):     by = self.names.index(by)
 
-        if values is None: values = "_"
+        if values is None:
+            values = "_"
+        else:
+            assert len(values) == len(self.columns), "Length of values does not match length of columns"
+            # convert string values to categorical num values
+            values2 = []
+            for i in range(0,len(values)):
+                if self.type(i) == "enum":
+                    values2.append(self.levels()[i].index(values[i]))
```

Comment:
i think for this release we should reject.
we can make a new jira for allowing new levels in Python or R client api, either through some other command or through impute